templates/themes/app/pagelayout/footer.html.twig

```diff
@@ -57,7 +57,10 @@
         </div>
 
         <div class="footer-info">
-            <a href="#" class="js-open-ng-cc d-block my-2">{{ 'ngsite.layout.cookie_settings'|trans }}</a>
+            {% if ngsite.siteInfoContent.hasField('related_cookie_policy') and not ngsite.siteInfoContent.fields.related_cookie_policy.empty %}
```

Comment:
Why do we need `hasField` call? We don't use it anywhere else in the template.

Reply:
unlike for icons, I have lost a bit of hair when someone deleted a cookie-policy field in siteinfo on live site and we had a huge problem :) 

Also, because of third party cookies, we clear up cookie policy content and fields much more often than favicon icons.

Up to you to decide, but lets unify this condition and the one in cookie_control.html.twig, whichever the decision will be.